install/playground/kubernetes/nginx.conf

```diff
@@ -36,30 +36,12 @@ http {
     include /etc/nginx/conf.d/*.conf;
 
     server {
-    if ($host = meshery-ws.layer5.io) {
+        listen 80;
+        server_name meshery-ws.layer5.io;
```

Comment:
```suggestion
        listen 80;
        listen [::]:80;
        server_name meshery-ws.layer5.io;
```

Reply:
Lets not remove the support for IPV6 addresses.

Reply:
Oh sorry. Did this by accident. Is there anything left to be done in the PR?